src/common/favicon.js

```diff
@@ -48,9 +48,9 @@ export const getBestIcon =
   const others = new Set();   // store icons without sizes or non-shortcut icons
 
   for (var icon of icons) {
-    if (icon.rel != 'shortcut icon') {
+    if (icon.rel != 'shortcut icon' && icon.rel != 'icon') {
       others.add(icon);
-    } else if (!icon.sizes) {
+    } else if (!icon.sizes || icon.sizes === "") {
```

Comment:
This shouldn't be required: empty strings are falsy, so this should be functionally identical to what you had before.